### DISCUSSION TITLE
Skinnable GamePlay HUD elements.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#15641] [Ideas] [answered]
This is more in reference to the Performance Points calculator, but could also apply to any other HUD element that is used in the HUD configuration screen (`CTRL + SHIFT + S`) and is not labeled as legacy.

Currently, [Performance Points Counter uses a predefined OsuFont ](https://github.com/ppy/osu/blob/83fea7158054abd584d8f0170be75833ba84db9f/osu.Game/Screens/Play/HUD/PerformancePointsCounter.cs#L193) without any methods to change the font or texture path of the Performance Points / Score / Time.

This can be a lower class priority as this could be used after a definitive method of lazer skinning is made.

**Comment by bdach** [comment, marked as answer]
this will come eventually with a new skinning system.

**Reply by Coppertine** [reply]
I'll keep this discussion open for when the new skinning system rolls around.
